### ISSUE TITLE
[BUG] import sql file for postgresql throws error

### ISSUE THREAD

**Issue by u007** [cweijan/vscode-mysql#678]
- OS: macos 12.5
- Database Type: postgresql

output:
```
export "PGPASSWORD=xxxxxx" && psql -h 127.0.0.1 -p 5432 -U postgres -d mydatabase -c 'set schema \'public\'' -f '/Users/username/Downloads/postgres_prices_2022-09-13_155159.sql'
/bin/sh: -c: line 0: unexpected EOF while looking for matching `''
/bin/sh: -c: line 1: syntax error: unexpected end of file
```

if i remove \'public\' to just public, it works

not sure whats the best delimiter for schema

**Comment by cweijan** [comment]
Thanks for your feedback, I will fix it in the next version.

**Comment by cweijan** [comment]
Hi, I have released version 5.7.12 to fix this problem.

**Comment by u007** [comment]
okay thank you
